subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidPluginsSmokeTest.groovy

```diff
@@ -77,9 +81,19 @@ class AndroidPluginsSmokeTest extends AbstractSmokeTest {
         """.stripIndent() << androidPluginConfiguration() << activityDependency()
 
         when:
-        def result = runner('androidDependencies', 'build', 'connectedAndroidTest', '-x', 'lint').build()
+        def result = runner(
+            'androidDependencies',
+            'build',
+            'connectedAndroidTest',
+            '--warning-mode', 'all',
```

Comment:
I think we should enable `--warning-mode=all` and the forwarding of output for all smoke tests, WDYT? I'll need it for my deprecation tests anyway.